pyvcloud/system_test_framework/constants/ovdc_network_constant.py

```diff
@@ -0,0 +1,19 @@
+# VMware vCloud Director Python SDK
+# Copyright (c) 2014-2019 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+class OvdcNetConstants(object):
+    routed_net_name = 'test-routed-vdc-network'
```

Comment:
Why do we need to add a new  constants file ?
make consistency with isolated and direct network.
base_config.xml
-------------------
  default_ovdc_network_name: 'test-isolated-vdc-network'
  default_ovdc_network_gateway_ip: '10.1.1.1'
  default_ovdc_network_gateway_netmask: '255.255.255.0'
  default_direct ovdc_network_name: 'test-direct-vdc-network'

Reply:
@kousgy123 Let's not add anything new to base_config.yaml. Anyway these value doesn't need to change base upon the environment. There is already a pending task on me to clear such records from the base_config.yaml and move to the constant files.

Reply:
Fine, I agree with you.